### PR TITLE
Change soil moisture to percent display

### PR DESCRIPTION
**Pull request by zuckschwerdt**
Changes Bresser 6in1 and Springfield soil moisture display to percent.
This is breaking (6in1 is new and probably not widely used).
See also #1492

### CODE REVIEW

**Comment by merbanan** [comment]
Normalisation is always good.

File: src/devices/bresser_6in1.c [hunk 1]
```diff
@@ -187,9 +187,9 @@ static int bresser_6in1_decode(r_device *decoder, bitbuffer_t *bitbuffer)
             "channel",          "",             DATA_INT,    chan,
             "battery_ok",       "Battery OK",   DATA_INT,    batt,
             "temperature_C",    "Temperature",  DATA_COND, temp_ok, DATA_FORMAT, "%.1f C", DATA_DOUBLE, temp_c,
-            "humidity",         "Humidity",     DATA_COND, temp_ok, DATA_INT,    humidity,
+            "humidity",         "Humidity",     DATA_COND, temp_ok && moisture < 0, DATA_INT,    humidity,
```
Comment (Mindavi) [inline src/devices/bresser_6in1.c:190]: Does this mean humidity is only shown when moisture < 0?

**Comment by Mindavi** [comment]
Other than the one question, diff LGTM

File: src/devices/bresser_6in1.c (hunk 1 above)
Comment (zuckschwerdt) [reply src/devices/bresser_6in1.c:190]: `moisture >= 0` is valid and thus no humidity should be shown (the moisture level is transmitted in the humidity field)

Comment (Mindavi) [reply src/devices/bresser_6in1.c:190]: Ah ok, clear. Good to know :)

**Pull request merged**